src/Microsoft.AspNetCore.Razor/Parser/TagHelpers/TagHelperBlockRewriter.cs

```diff
@@ -229,6 +238,15 @@ public static class TagHelperBlockRewriter
                     // Check for attribute start values, aka single or double quote
                     if (i < htmlSymbols.Length && IsQuote(htmlSymbols[i]))
                     {
+                        if (htmlSymbols[i].Type == HtmlSymbolType.DoubleQuote)
+                        {
+                            attributeValueStyle = HtmlAttributeValueStyle.DoubleQuotes;
```

Comment:
Why bother i.e. why not make changes only in the single quotes and minimized cases?